extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -627,8 +633,13 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
         }
         ulHtml += '</li>';
       }
+      this.preventDisplaying = false;
       if (this.contactSearchInProgress) {
-        ulHtml += '<li class="loading" data-test="container-contacts-loading">loading...</li>';
+        if (this.canSearchContacts) {
+          this.preventDisplaying = true;
+        } else {
+          ulHtml += '<li class="loading" data-test="container-contacts-loading">loading...</li>';
```

Comment:
It seems counter intuitive that the loader would stay there exactly when user cannot search contacts. Is this to have some space to put the new permission prompt into?

Reply:
I believe the naming of the variable is confusing here. Maybe it should be `googleContactsSearchEnabled`?

The loading state is needed for searching contacts from the storage (locally):

![CleanShot 2021-07-26 at 22 51 18](https://user-images.githubusercontent.com/6059356/127050090-2559f52c-a2e6-4aaf-b621-53397a639733.gif)

But now, that I'm looking at it and the "loading..." state is so quick that it looks like a glitch, should we remove the loading state completely for both local search and google contacts search?

Reply:
That's what I was thinking - remove both, since local should be fast

Reply:
Done 👍